app/views/shared/_summary_lgfs.html.haml

```diff
@@ -16,7 +16,7 @@
               = t('.fee_category')
             %th{ scope: 'col' }
               = t('.fee_type')
-            - if claim.transfer? || (claim.final? && !claim.fixed_fee_case?) # could be applied to graduated too
+            - if claim.transfer? || (claim.final? && !claim.fixed_fee_case?)
```

Comment:
👍 cool, as discussed you can merge and then do a separate PR to move this conditional logic into presenter(s) - employing either inheritance in fee presenters(s) or placing logic in `base_claim_presenter`